### PR TITLE
Added Contact Field in Company Registration.

### PR DESCRIPTION
**Pull request by KD-3**
### Description
Modified form.py, views.py and models.py of accounts app and register.html

### Type of Change:

- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- New feature (non-breaking change which adds functionality pre-approved by maintainers)

### Checklist:

- [x] My PR follows the style guidelines of this project
- [x] Any dependent changes have been merged 

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/accounts/forms.py [hunk 1]
```diff
@@ -76,6 +76,7 @@ class CompanyRegisterForm(UserCreationForm):
     state = forms.CharField(max_length=15, required=False)
     country = forms.ChoiceField(choices=CompanyProfile.NATION)
     pin_code = forms.CharField(max_length=10, required=False)
+    contact = forms.CharField(max_length=20, required=False)
```
Comment (anshulahuja98) [inline src/accounts/forms.py]: Make `required=True`

File: src/accounts/models.py [hunk 2]
```diff
@@ -67,6 +67,7 @@ class CompanyProfile(models.Model):
                                         through_fields=('company', 'student'), related_name='joboffers')
     internship_offers = models.ManyToManyField(StudentProfile, through='company.InternshipOffer',
                                                through_fields=('company', 'student'), related_name='internshipoffers')
+    contact = models.CharField(max_length=20, null=True, blank=True)
```
Comment (anshulahuja98) [inline src/accounts/models.py]: Remove `null=True` and `blank=True`

File: src/templates/company/register.html [hunk 3]
```diff
@@ -100,6 +100,14 @@
                                 <label for="id_pin_code">Pin code:</label>
                                 <input type="text" name="pin_code" maxlength="10" id="id_pin_code"/>
                             </div>
+                            <br>
+                            <div class="md-form">
+                                <i class="fa fa-map-marker prefix" style="color: cornflowerblue"></i>
```
Comment (anshulahuja98) [inline src/templates/company/register.html]: These are fontawesome icon, use a phone icon instead

File: src/templates/company/register.html [hunk 4]
```diff
@@ -100,6 +100,14 @@
...
+                                <label for="id_contact">Contact:</label>
+                                <input type="text" name="contact" maxlength="20" id="id_contact"/>
```
Comment (anshulahuja98) [inline src/templates/company/register.html]: I hope you have tested the form once?

Comment (KD-3) [reply src/templates/company/register.html]: Yes.

**Pull request merged**